config/samples/secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    argocd.argoproj.io/secret-type: cluster
```

Comment:
just a thought for the future but prob should add our own label to secrets to indicate which clusters users want us to manage ingress for?

Reply:
I did it this way, as this is how Argo labels it's secrets for clusters. Would we expect a user of Argo to go and add our label to the secrets created by Argo?

Reply:
I think this is not a bit problem from my understanding you can create the secrets declaratively but not an issue right now. I think though we would also want them to add annotations such as geo and cloud provider at somepoint

Reply:
It sounds reasonable for the controller to only look for secrets with a specific label for mctc.
It would avoid a dependency on the labelling that ArgoCD expects and allow some clusters to be omitted from mctc but still used by ArgoCD